GoogleDataTransport/GDTCORLibrary/Private/GDTCORUploadBatch.h

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
In the rest of the repo, the Private folder is for internal headers used by other libraries in the repo.

Headers only needed by their own library are typically in the same folder as the sources.

We should consider making GDT consistent with that, either as new headers are added or as a follow on.

Reply:
I'll follow common private/internal headers rules for the future headers. I think the current ones can be moved as a part of #6383. I added a note to the ticket description.